### ISSUE TITLE
[Feature Request] render embedded markdown link as block instead of an hyperlink

### ISSUE THREAD

**Issue by sasasqt** [shd101wyy/vscode-markdown-preview-enhanced#1746]
**Is your feature request related to a problem? Please describe.**

```
![[rl#td-error-vs-advantage-vs-bellman-error]]
```
instead of rendering it as a hyperlink
![image](https://github.com/shd101wyy/vscode-markdown-preview-enhanced/assets/56587914/6341ca1a-80ea-4225-b394-03f375d2e1f0)
i wish it to mimic the behaviour in dendron
dendron renders it as: 
![image](https://github.com/shd101wyy/vscode-markdown-preview-enhanced/assets/56587914/33ec1878-38e8-47b7-8721-8a4e7d01c991)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by shd101wyy** [comment]
Yes that's also what I am planning to support. We now have the syntax:

```
@import "file.md"
```

we will also support `![]()` to embed files.

**Comment by sasasqt** [comment]
is it possible to only import some sections/from line x to line y of file.md?

some thing likes: @import "file.md#section1"
some thing likes: @import "file.md#linex-liney"
